courses-app/src/App.jsx

```diff
@@ -21,8 +22,9 @@ function App() {
               coursesList={coursesList}
               setIsCreateOpened={setIsCreateOpened}
               isCreateOpened={isCreateOpened}
+              setFilteredList={setFilteredList}
             />
-            <Courses courses={coursesList} setCoursesList={setCoursesList} />
+            <Courses courses={filteredList} setCoursesList={setCoursesList} />
```

Comment:
Is there a need to pass the **setCoursesList** to Courses component? Please remove it if it doesn't need